api/models.js

```diff
@@ -159,8 +159,8 @@ var projectSchema = mongoose.Schema({
     },
 
     quota: {type: Number, default: 1000000000000}, //maximum archive size (1TB by default)
-
-    /*
+    mag: {
+      },    /*
```

Comment:
Missing schema?

Reply:
Please be sure to populate all object/fields down to the leaf elements. We need to make sure we aren't adding things unexpectedly.